math.go

```diff
@@ -37,18 +37,18 @@ func newPairEntry(data1, data2 uint32) pairEntry {
 
 // vec3I represents a 3D vector typed as int32
```

Comment:
Comment should be updated to int64. I’d also double-check if any other comments need to be updated too